Neos.Fusion/Classes/Core/Cache/ContentCache.php

```diff
@@ -371,6 +370,17 @@ protected function getTypeForContextValue($contextValue)
         return $type;
     }
 
+    /**
+     * Get a content cache entry by it's identifier
+     *
+     * @param string $cacheIdentifier
+     * @return mixed The cache entry's content as a string or false if the cache entry could not be loaded
+     */
+    public function getCacheByIdentifier(string $cacheIdentifier)
```

Comment:
Wouldn't it be possible to to add a `cacheIdentifierIsWasRequested(string $identifier, bool $hit)`to monitor cache hits?
If not, I would prefer a protected method here.

Reply:
Would be possible, sure. But then you'd add a new empty method?

Reply:
But as a signal it would be a planned extension point / API. The protected method could likely be refactored and get lost sooner or later.

Reply:
Ok, i'll add a signal